test/integration/editor/new.spec.ts

```diff
@@ -77,3 +76,20 @@ test('can create/delete page', async ({ page, localApp }) => {
 
   await expect.poll(async () => folderExists(pageFolder)).toBe(false);
 });
+
+test('can rename page', async ({ page, localApp }) => {
+  const editorModel = new ToolpadEditor(page);
+
+  await editorModel.goto();
+
+  await editorModel.createPage('someOtherPage');
+
+  const pageFolder = path.resolve(localApp.dir, './toolpad/pages/someOtherPage');
```

Comment:
Just to protect ourselves from accidentally breaking this test, let's double verify the folder exists
```suggestion
  const pageFolder = path.resolve(localApp.dir, './toolpad/pages/someOtherPage');
  await expect.poll(async () => folderExists(pageFolder)).toBe(true);
```